core/store/src/adapter/trie_store.rs

```diff
@@ -167,3 +194,103 @@ pub fn get_key_from_shard_uid_and_hash(shard_uid: ShardUId, hash: &CryptoHash) -
     key[8..].copy_from_slice(hash.as_ref());
     key
 }
+
+#[cfg(test)]
+mod tests {
+    use assert_matches::assert_matches;
+    use near_primitives::errors::StorageError;
+    use near_primitives::hash::CryptoHash;
+    use near_primitives::shard_layout::ShardUId;
+
+    use crate::adapter::trie_store::TrieStoreAdapter;
+    use crate::NodeStorage;
+
+    const ONE: std::num::NonZeroU32 = match std::num::NonZeroU32::new(1) {
```

Comment:
hehe that is a fun way to write 1

nit: Can you do unwrap instead of match?

Reply:
I wanted but compiler complains :man_shrugging: 

> `std::option::Option::<T>::unwrap` is not yet stable as a const fn

Reply:
`lib.rs` already deal with this using match, so I copied them.